tests/governance.py

```diff
@@ -592,6 +593,41 @@ def test_all_nodes_cert_renewal(network, args, valid_from=None):
             ), f"Self-signed node certificate for node {node.local_node_id} was not renewed"
 
 
+@reqs.description("Test COSE Sign1 auth")
+def test_cose_auth(network, args):
+    primary, _ = network.find_primary()
+    identity = primary.identity("member0")
```

Comment:
Not really related to this PR, but it's odd this is on `Node` rather than `Network`. Or maybe a free function taking `common_dir` as an arg in `path.py`, with `Node.identity()` using `Node.common_dir` and external callers using `network.common_dir`? Very minor.

Reply:
The identity you mean? Yes, agreed, I just used what was there, but it would be more sensible for it to be on the network. Will change.